### PR TITLE
links are unavailable

### PR DESCRIPTION
**Pull request by tim-zju**
Signed-off-by: tim-zju <21651152@zju.edu.cn>

**- What I did**
delete links which are unavailable in docs->api->version-history.md
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/api/version-history.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 ## v1.26 API changes
 
-[Docker Engine API v1.26](v1.26/) documentation
+[Docker Engine API v1.26] documentation
 
 ## v1.25 API changes
 
-[Docker Engine API v1.25](v1.25.md) documentation
```
Comment (thaJeztah) [inline docs/api/version-history.md]: I think this will become https://docs.docker.com/engine/api/v1.25/ once 1.13 is released (same for the v1.26 above)

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
Please squash commits

File: docs/api/version-history.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 ## v1.26 API changes
 
-[Docker Engine API v1.26](v1.26/) documentation
+[Docker Engine API v1.26](v1.26.md) documentation
```
Comment (thaJeztah) [inline docs/api/version-history.md]: Can you change this to https://docs.docker.com/engine/api/v1.26/, and the v1.25 link to  https://docs.docker.com/engine/api/v1.26/ ?

The documentation for docker 1.13 and 1.14 is not published yet, so the links will not be there yet, but once docker they're up, those should be the URL's. See https://github.com/docker/docker.github.io/tree/vnext-engine/engine/api/v1.25

**Comment by vdemeester** [comment]
ping @tim-zju

**Comment by thaJeztah** [comment]
closing, because there was no response, but let us know if you still want to work on this

**Pull request closed without merging**